src/services/ol-layer/ol-layer-feature-position.helper.ts

```diff
@@ -0,0 +1,60 @@
+import { Geometry, Point } from 'ol/geom'
+import VectorLayer from 'ol/layer/Vector'
+import { Options } from 'ol/layer/BaseVector'
+import VectorSource from 'ol/source/Vector'
+import Feature from 'ol/Feature'
+
+import { OlLayer } from './ol-layer.model'
+import { getStyleFeaturePosition } from './styles.helper'
+
+export const DEFAULT_LAYER_ZINDEX = 10000
+export const FEATURE_LAYER_TYPE = 'featurePositionLayer'
+
+export class PositionVectorLayer extends VectorLayer<VectorSource<Geometry>> {
+  geoMarker: Feature<Point> | undefined
+
+  constructor(options: Options<VectorSource<Geometry>>) {
+    const source = new VectorSource({
+      features: [], // geoMarker to be added here (@see createGeomarker())
+    })
+
+    super({ ...options, ...{ source } })
+  }
+
+  moveGeoMarker(x: number, y: number) {
```

Comment:
Just a detail, but the case of `...Marker` is not the same as in the following methods.

Reply:
fixed